scripts/push_images.sh

```diff
@@ -88,10 +88,12 @@ if [ $QUIET = true ]; then
   )
 fi
 
+date_str=$(date +%Y%m%d)
```

Comment:
If this runs before midnight this will fail when the date changes

Reply:
The workflow is set to run at "0 2 * * 1" so this shouldn't be an issue.
Maybe we an add a `--utc` though?

Reply:
Alternatively, you could avoid this altogether by not tagging the image with date in build_images.sh, instead creating the dated tag from the just built and well-known "latest" (e.g. "ubuntu") tag right before pushing in the push_images.sh script

Reply:
Reading the [`depot push` documentation](https://depot.dev/docs/cli/reference#depot-push) a little more closely, the logic appears to be more similar to the following:

```
docker tag $LOCAL_IMAGE@sha256:xxx $REMOTE_REPO/$REMOTE_IMAGE:$REMOTE_TAG
docker push $REMOTE_REPO/$REMOTE_IMAGE:$REMOTE_TAG
```

instead of

```
docker tag $LOCAL_IMAGE:$LOCAL_TAG $REMOTE_REPO/$REMOTE_IMAGE:$REMOTE_TAG
docker push $REMOTE_REPO/$REMOTE_IMAGE:$REMOTE_TAG
```

The depot build ID should be stable across dates.